### PR TITLE
Update APAC frequency to bi-weekly

### PR DESCRIPTION
**Pull request by 0x646e78**
Have also added a link to the CNCF calendar for people to be able to check the date of the biweekly meeting.

Decision documented [here](https://docs.google.com/document/d/170y5biX9k95hYRwprITprG6Mc9xD5glVn-4mB2Jmi2g/edit#heading=h.eovac03mwb5r)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,10 +45,12 @@ Anyone is welcome to join our open discussions of SIG-Security projects and shar
 
 ## Meeting times
 
-The Security SIG group meets every week.
+The Security SIG group meets regularly.
```
Comment (TheFoxAtWork) [inline README.md:48]:
```suggestion
The Security SIG group meets at least weekly.
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
one small change.  otherwise LGTM!

File: README.md [hunk 2]
```diff
@@ -45,10 +45,12 @@ Anyone is welcome to join our open discussions of SIG-Security projects and shar
...
 
-* US:   Wednesdays at 10:00am GMT-7 (see your timezone [here](https://time.is/compare/1000_27_Jan_2021_in_PT))
-* APAC: Tuesdays at 13:00pm GMT+11 (see your timezone [here](https://time.is/compare/1300_26_Jan_2021_in_AEDT))
+* US:   Weekly on Wednesdays at 10:00am GMT-7 (see your timezone [here](https://time.is/compare/1000_27_Jan_2021_in_PT))
+* APAC: Bi-weekly on Tuesdays at 13:00pm GMT+11 (see your timezone [here](https://time.is/compare/1300_26_Jan_2021_in_AEDT))
```
Comment (lumjjb) [inline README.md:51]: 13:00pm -> 1:00pm.

**Review by lumjjb — CHANGES_REQUESTED** [review]
minor nit, if not LGTM too

File: README.md (hunk 2 above)
Comment (dnx-seek) [reply README.md:51]: Yes I totally agree! I will update.

File: README.md (hunk 1 above)
Comment (0x646e78) [reply README.md]: Pushed an update to reflect this.

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

**Comment by 0x646e78** [comment]
@lumjjb looks OK now?

**Review by ultrasaurus — APPROVED** [review]
I don’t think “at least weekly” is the right way to say it, since we sometimes cancel on holidays, but I do think this is important to merge so APAC times are correct before next week!

File: README.md [hunk 3]
```diff
@@ -45,10 +45,12 @@ Anyone is welcome to join our open discussions of SIG-Security projects and shar
 
 ## Meeting times
 
-The Security SIG group meets every week.
+The Security SIG group meets at least weekly.
```
Comment (ultrasaurus) [inline README.md:48]:
```suggestion
Group meeting times are listed below:
```

**Review by lumjjb — APPROVED** [review]
LGTM

**Comment by pragashj** [comment]
Closing this issue as a result of this merge. https://github.com/cncf/sig-security/issues/312

**Pull request merged**